multiqc/modules/odgi/odgi.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from odgi stats """
+
+from __future__ import print_function
+import logging
+from collections import OrderedDict
+
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import linegraph
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    """
+    The MultiQC module to parse and plot odgi stats output.
+    """
+
+    def __init__(self):
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Odgi', anchor='odgi',
+                                            href='https://github.com/vgteam/odgi',
+                                            info='is an optimized dynamic graph/genome implementation.')
+
+        self.odgi_stats_map = dict()
+
+        # Find and load any odgi stats file
+        log_files = self.find_log_files('odgi')
+        if not log_files:
+            raise UserWarning
```

Comment:
We are raising one here @ewels ?

Reply:
You need to use `self.ignore_samples()` too: https://multiqc.info/docs/#filtering-by-parsed-sample-names

Reply:
See Kallisto for an example of this: https://github.com/ewels/MultiQC/blob/735f0bc57f408d0147a81cad1e93a711a84328e9/multiqc/modules/kallisto/kallisto.py#L31-L42